README.md

```diff
@@ -100,12 +106,27 @@ Before you get started, the following needs to be installed:
   bundle exec rake jobs:work
   ```
 
-1. In a new console, open the project root folder and start the server. The simplest way is to use the included Webrick server:
+
+1. Starting the development server in a new console (open the project root folder)
+Some components are created with React and they need to be build with foreman
+
+  1. Hot Reloading of Rails Assets
 
   ```bash
-  bundle exec rails server
+  foreman start -f Procfile.hot
   ```
 
+  1. Static Loading of Rails Assets
+  ```bash
+  foreman start -f Procfile.static
+  ```
+
+  1. Creating Assets for Tests
+  ```bash
+  foreman start -f Procfile.spec
+  ```
+
```

Comment:
As somebody not familiar with React, hot loading and react-on-rails this section is not very clear. Should I invoke all those commands one time? Every time I change something? Do they start a continuous server processes to do something every time a change is made?